lib/_http_common.js

```diff
@@ -21,7 +21,11 @@
 
 'use strict';
 
-const { methods, HTTPParser } = internalBinding('http_parser');
+const { getOptionValue } = require('internal/options');
+
+const { methods, HTTPParser } =
+  getOptionValue('--http-parser') === 'traditional' ?
```

Comment:
There’s only two places where we actually do the comparison/switch, one in a test, so I don’t really think this is an issue (i.e. this is already the “central place” to make changes, I’d say). If that’s too much, we could make `HTTPParser` monkey-patchable in this module, because that’s what the test does.

I didn’t choose a boolean, because it gets somewhat icky having multiple boolean flags (e.g. `--http-parser-legacy` and `--http-parser-llhttp`) that conflict; and I think we’d want flags for enabling either parser explicitly, at least for some time.